tox.ini

```diff
@@ -12,5 +13,5 @@ setenv =
 # The standard library unittest module can run tests on Python 2.7 and newer
 commands =
     python -c "import OpenSSL.SSL; print(OpenSSL.SSL.SSLeay_version(OpenSSL.SSL.SSLEAY_VERSION))"
-    coverage run --branch --source=OpenSSL setup.py test
+    coverage run --branch --source=OpenSSL -m pytest OpenSSL
```

Comment:
I will make `setup.py test` work again once I’ve verified that the test suite actually works.